guava/src/com/google/common/hash/HashCode.java

```diff
@@ -82,22 +89,27 @@
    * @return the number of bytes written to {@code dest}
    * @throws IndexOutOfBoundsException if there is not enough room in {@code dest}
    */
+  @SuppressWarnings({"lowerbound:assignment.type.incompatible",// Since bits() return non negative value, `bits() / 8`
```

Comment:
Why isn't `bits()` annotated to return non-negative, then? The checker should be able to infer that `@NonNegative / 8` is also `@NonNegative`